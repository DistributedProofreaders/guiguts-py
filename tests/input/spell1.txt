

Next paragraph. sd asd askd.
