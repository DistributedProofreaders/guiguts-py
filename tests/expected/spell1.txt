Start of Spelling Check

3.16: sd (1)
3.19: asd (1)
3.23: askd (1)

Check complete

